playground/backend/pkg/executors/java_executor.go

```diff
@@ -0,0 +1,64 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package executors
+package executors
+
+import (
+	"beam.apache.org/playground/backend/pkg/api/pathchecker"
+	"beam.apache.org/playground/backend/pkg/fs_tool"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	beamJarPath   = "/opt/apache/beam/jars/beam-sdks-java-harness.jar"
+	runnerJarPath = "/opt/apache/beam/jars/beam-runners-direct.jar"
+	slf4jPath     = "/opt/apache/beam/jars/slf4j-jdk14.jar"
+)
+
+// JavaExecutor for Java code
+type JavaExecutor struct {
+	Lang string
+	fs   fs_tool.JavaFileSystemService
+}
+
+// Validate checks that the file exists and that extension of the file matches the SDK
```

Comment:
I'm not sure but maybe we don't need to write comments for the interface's method and for the implementation of this method. We could write it only for the interface's methods. @ilya-kozyrev @KhaninArtur what do you think about it?

Reply:
I agree, let's keep them just in interfaces.